clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -114,7 +114,7 @@
         "test": "jest -w 1"
     },
     "dependencies": {
-        "@zowe/imperative": "^4.4.5",
```

Comment:
@ishche should it be here, what do you think? As well as `package-lock.json` changes?

Reply:
Too keep version fixed, otherwise needs another TPSR

Reply:
Is not clear for me why it was removed from the package json..

Reply:
@zacanbrcom  Not removed, just fixed version used,  "@zowe/imperative": "4.4.5" because of TPSR

Reply:
Got it! TY